dev.py

```diff
@@ -166,8 +165,24 @@ def main(argv):
                         "Note: this argument may be removed in the future "
                         "once a `site.cfg`-like mechanism to select BLAS/LAPACK "
                         "libraries is implemented for Meson")
+    parser.add_argument("--build-dir", default="build",
+                        help="Relative path to the build directory. "
+                             "Default is 'build'")
+    parser.add_argument("--install-prefix", default=None,
+                        help="Relative path to the install directory. "
+                             "Default is <build-dir>-install.")
     args = parser.parse_args(argv)
 
+    global PATH_INSTALLED
+    build_dir = Path(args.build_dir)
```

Comment:
Worth checking that `build_dir` and `install_dir` do not already exist, or are previous build / installs?    For example, what should happen for `python dev.py --build-dir scipy`?

Reply:
Since this is going to be used mostly by devs, I don't think it's worth the extra effort. Something like `python dev.py --build-dir scipy` is clearly a bad practice.

Reply:
Surely not that much effort to check, for example:

```
if build_dir.exists() and not (build_dir / 'meson_info').exists():
```

etc?    It's not all that hard to imagine, for example, `python dev.py --build-dir tools` or `python dev.py --build-dir benchmarks`.

Reply:
OK, I think it'd be fine to throw an exception if one tries to build/install into non-empty directories. Does it look good?

Reply:
LGTM